laphayen/VisitorQueueSystem
# Create a Spring MVC project for the target web page

### Key Updates:

Converted configuration from application.properties to application.yml for improved structure and readability.
Set the server port to 9000 in application.yml.
Introduced a new index.html file as the primary landing page with a responsive, user-friendly design.
Configured the root mapping to return "index", linking it to the index.html landing page.
Added root mapping (@GetMapping("/")) in VisitorQueueSystemWebsiteApplication to handle requests to the root URL.

* * *

### How to Test:

Run the Spring Boot application.
Open a web browser and navigate to http://localhost:9000/.
Verify that the landing page (index.html) is displayed with the title, description, and call-to-action button.

* * *

This closes #1
